### PR TITLE
[ES] Add more sentences to test getting the temperature

### PR DESCRIPTION
**Pull request by cibernox**
Particularly I added masculine examples and examples that used "de la / del" instead of "en la / en el", which is also a valid way command that wasn't tested.

### CODE REVIEW

**Review by cibernox — COMMENTED** [review]
Somewhat unrelated, but I figured out why `¿Cual es la temperatura del salón?` didn't work for me. It was because my living room didn't have a thermostant entity on it, it had a temperature sensor instead.

Turns out temperature sensors are not valid targets for temperature queries, and I don't think that's right. It doesn't seem exclusive to spansh language. I have temperature sensors in my pool and in my greenhouse, none if which will ever have a thermostat. That seems like it should change.

**Comment by davefx** [comment]
> Somewhat unrelated, but I figured out why `¿Cual es la temperatura del salón?` didn't work for me. It was because my living room didn't have a thermostant entity on it, it had a temperature sensor instead.
> 
> Turns out temperature sensors are not valid targets for temperature queries, and I don't think that's right. It doesn't seem exclusive to spansh language. I have temperature sensors in my pool and in my greenhouse, none if which will ever have a thermostat. That seems like it should change.

About this, I would open a different issue, as it's unrelated to the language.

File: shared/__init__.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from dataclasses import dataclass, field
 from datetime import datetime
 from functools import partial
-from typing import Any, Dict, List, Optional, Set, Tuple
+from typing import Any, Dict, Optional, List, Set, Tuple
```
Comment (cibernox) [inline shared/__init__.py:4]: FWIW, this was a result of running the linting. I agree it's unrelated but it's possibly something desirable

**Comment by cibernox** [comment]
@davefx I opened this issue: https://github.com/home-assistant/intents/issues/2469

I'm still not sure if this belongs on this repo or on HA Core, but someone will tell me either way.

**Review by davefx — APPROVED** [review]

**Pull request merged**